test/e2e/operator_test.go

```diff
@@ -462,9 +462,11 @@ func TestExternalDNSCustomIngress(t *testing.T) {
 	}
 
 	openshiftRouterName := "external-dns"
-	name := types.NamespacedName{Namespace: testIngressNamespace, Name: openshiftRouterName}
+	// ingress controllers are supposed to be created in the ingress operator namespace
+	name := types.NamespacedName{Namespace: "openshift-ingress-operator", Name: openshiftRouterName}
+	ingDomain := name.Name + "." + hostedZoneDomain
```

Comment:
Can you change this to a string format instead.

Reply:
Don't want to start a holy war but generally I prefer the standard concatenation over any formatting function for the small and simple patterns like here. Because it's reported as being faster, by quite some sources (at least it used to be by the time I took this habit).
However in this file I see the total usage of `Sprintf`, so changing it just to be coherent.